src/style.css

```diff
@@ -0,0 +1,70 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
leave only box-sizing here. Margin and padding reset better to move to 'body'

Reply:
@varseniuk  if I move it to <body>, everything is ruined... have no idea why :(

Reply:
That's because you should reset padding and margin not only in 'body', but also in your 'ul'. The '*' selector resets all paddings and margins, and sometimes it may be unuseful and unpredictable.
But ok, for this task let it be